File: RepoTransBench/target_projects/C/Python/mbuesch_razer/tests/original/test_buttonmapping.py
import pytest

BUTTON_MAPPING_SIZE = 4

class ButtonMap:
    def __init__(self):
        self.mapping = [0] * BUTTON_MAPPING_SIZE

def test_buttonmapping():
    bm = ButtonMap()
    for i in range(BUTTON_MAPPING_SIZE):
        bm.mapping[i] = i
    for i in range(BUTTON_MAPPING_SIZE):
        assert bm.mapping[i] == i

File: RepoTransBench/target_projects/C/Rust/mbuesch_razer/tests/original/test_buttonmapping.rs
#[test]
fn test_buttonmapping() {
    const BUTTON_MAPPING_SIZE: usize = 4;

    struct ButtonMap {
        mapping: [i32; BUTTON_MAPPING_SIZE],
    }

    let mut bm = ButtonMap {
        mapping: [0; BUTTON_MAPPING_SIZE],
    };
    for i in 0..BUTTON_MAPPING_SIZE {
        bm.mapping[i] = i as i32;
    }
    for i in 0..BUTTON_MAPPING_SIZE {
        assert_eq!(bm.mapping[i], i as i32);
    }
    println!("test_buttonmapping passed");
}